<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 107, in main
    result = linter.lint(
             ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 620, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 684, in lint_file
    recipe = _recipe.Recipe.from_file(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/recipe.py", line 340, in from_file
    raise exc
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/recipe.py", line 331, in from_file
    recipe.load_from_string(text.read())
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/recipe.py", line 214, in load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/recipe.py", line 421, in render
    self.meta = yaml.load(yaml_text)
                ^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/main.py", line 434, in load
    return constructor.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 119, in get_single_data
    node = self.composer.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 76, in get_single_node
    document = self.compose_document()
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 99, in compose_document
    node = self.compose_node(None, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 143, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 223, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 143, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 223, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 141, in compose_node
    node = self.compose_sequence_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 184, in compose_sequence_node
    while not self.parser.check_event(SequenceEndEvent):
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/parser.py", line 146, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/parser.py", line 530, in parse_block_sequence_entry
    if self.scanner.check_token(BlockEntryToken):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/scanner.py", line 1794, in check_token
    while self.need_more_tokens():
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/scanner.py", line 211, in need_more_tokens
    self.stale_possible_simple_keys()
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/scanner.py", line 360, in stale_possible_simple_keys
    raise ScannerError(
ruamel.yaml.scanner.ScannerError: while scanning a simple key
  in "<unicode string>", line 62, column 1:
    q
    ^ (line: 62)
could not find expected ':'
  in "<unicode string>", line 63, column 41:
     ... st -f $PREFIX/lib/libicui18n.a  # [not win]
                                         ^ (line: 63)

ERROR conda.cli.main_run:execute(47): `conda run conda-lint /tmp/abs_8fqk8p_o0k/clone` failed. (See above for error)
</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,2 +1,6 @@
 channels:
   icu73: icu73
+
+build_parameters:
+  - "--suppress-variables"
+  - "--error-overlinking"

